tool/structgen/src/main/java/org/openscience/cdk/structgen/maygen/Maygen.java

```diff
@@ -2594,21 +2590,8 @@ public void structureGenerator(String localFormula) {
         List<int[]> newDegrees = distributeHydrogens();
 
         if (multiThread) {
-            try {
-                new ForkJoinPool(size)
```

Comment:
size is important, it can't be removed.

Reply:
Please explain, I asked @MehmetAzizYirik. My understanding this code is not doing anything since the parellel stream will not use the forjoin pool

Reply:
Okay see here it really does work that way but is quite opaque/brittle - https://stackoverflow.com/questions/30802463/how-many-threads-are-spawned-in-parallelstream-in-java-8. "It should be noted that the Stream APIs use of ForkJoinPool is an implementation detail. Thus, both solutions work with Oracle’s current implementation but are not guaranteed to work everywhere."

Reply:
Why size and not - 
```java
ForkJoinPool pool = new ForkJoinPool(Runtime.getRuntime().availableProcessors());
```

Reply:
And that is the default anyways for ForkJoinPool in ParellelStream which is cleaner IMO.

Reply:
@johnmay we had that conversation with Valentyn about the size parameter, but I believe this needs to be changed like you explained we can use that other option you recommended. 

Also, this parallelization actually needs to be enhanced but further tasks not for now. Thus, we dont need to keep it with the ForkJoinPool but can change it later.

Reply:
> Please explain, I asked @MehmetAzizYirik. My understanding this code is not doing anything since the parellel stream will not use the forjoin pool

It is related for the bug https://github.com/MehmetAzizYirik/MAYGEN/issues/48

In some configuration default value is 79. We need to control it not use default.